scripts/validate/check-all-packages-vendored

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+set -eu
+
+FILE_PATH='/tmp/vndr.log'
+WARN_MISS_VENDOR='WARNING: dependency is not vendored'
+
+if grep -q $WARN_MISS_VENDOR $FILE_PATH; then
+	echo 
+	echo "Some dependencies are not vendored."
```

Comment:
Would it be difficult to print which dependencies? Perhaps we can print the output of those warning lines?

Reply:
umm, the previous run bash command already prints these lines.
bash -c 'vndr |& grep -v -i clone | tee /tmp/vndr.log'
But these lines are intertwined with lot of other logs as well. 
We can add the warning lines again or perhaps ask the user to look at the log file.
What do you think?